aries-test-harness/features/0160-connection.feature

```diff
@@ -85,7 +85,7 @@ Feature: RFC 0160 Aries agent connection functions
       When "Mallory" sends a connection request to "Acme" based on the connection invitation
       Then "Acme" sends a request_not_accepted error
 
-   @T004-AIP10-RFC0160 @SingleUseInvite @P2 @normal @ExceptionTest @WillFail @OutstandingBug..418..https://github.com/hyperledger/aries-cloudagent-python/issues/418 @allure.issue:https://github.com/hyperledger/aries-cloudagent-python/issues/418 @RFC0160
+   @T004-RFC0160 @SingleUseInvite @AIP10 @normal @ExceptionTest @WillFail @OutstandingBug..418..https://github.com/hyperledger/aries-cloudagent-python/issues/418 @allure.issue:https://github.com/hyperledger/aries-cloudagent-python/issues/418 @RFC0160
```

Comment:
One thing I noticed in seeing all the tags was that there were multiple formats for the "issue-related" tag -- in some cases "issue:<URL>" and then this format with "@OutstandingBug..".  I thought it would be good to use one predictable format.

Reply:
This is a good point. The WillFail and OutstandingBug are used in code and output this info in the test results when using the normal Behave output. That allure.issue tag is new and puts a clickable url in the Allure reports.  I could check for the allure.issue tag now and remove that OutstandingBug tag.

Looking at this now, raised the question, this test may not fail when running on another agent. These WillFail and Issue tags will not correspond to a passing agent. Not sure how to handle this. Will have stew on that some more.

Reply:
One thing we could do -- have a tag format of issue{:<agent>}:<number>, where the agent is optional.  That way, we could just use "acapy" or "afgo" to indicate that the issue is in those repos.  If not present, the issue is in AATH.  So examples:

@issue:acapy:418
@issue:418

The <agent> would use the same format as the .manage script.

Reply:
So this means the `@allure.issue:https...` tag must stay as is (can't put an agent in that tag or allure won't find it).
We will remove `@WillFail` & `@OutstandingBug..XXX..http...`. 
We will replace those two with `@issue:acapy:XXX`, or `@issue:XXX`.
The removal of `@WillFail` means that we won't list issues on tests unless they cause the test to fail. A test with an issue that doesn't fail has not happened yet, but I envisioned it when I created the `@WillFail` tag.

Reply:
Cut the issue tags down to just the allure.issue one. If that exists, then there is a message sent to tell users that this will fail because of this issue.